superset/assets/src/visualizations/Table/Table.js

```diff
@@ -130,6 +130,12 @@ function TableVis(element, props) {
     'table-condensed table-hover dataTable no-footer', true)
     .attr('width', '100%');
 
+ 
+  function buttoRenderer(label) {
+      // console.log(slice.datasource)
```

Comment:
(nit) commented code should not be merged into master

Reply:
done